src/focus.service.ts

```diff
@@ -421,6 +425,9 @@ export class FocusService {
    * Frees resources associated with the service.
    */
   public teardown() {
+    if (!this.registrySubscription) {
+      return;
+    }
     this.registrySubscription.unsubscribe();
```

Comment:
Does this work? `this.registrySubscription ? this.registrySubscription.unsubscribe() : return;`

Reply:
no.